File: ffmpeg-playground/fix-jsx.js
const fs = require('fs');
const path = require('path');

const filePath = path.join(__dirname, '../video-editor/app/playground/orchestration/[projectId]/page.tsx');

let content = fs.readFileSync(filePath, 'utf8');

// Find the line with </button> followed by )}
const pattern = /(\s+<\/button>\r?\n)(\s+)\)\}/;
const replacement = '$1$2  </>\r\n$2)}';

content = content.replace(pattern, replacement);

fs.writeFileSync(filePath, content, 'utf8');
console.log('Fixed JSX syntax error - added closing fragment tag');

File: fix-jsx.py
import re

file_path = r'c:\Users\USER\Desktop\desktop\test\lumina-video-agent\video-editor\app\playground\orchestration\[projectId]\page.tsx'

with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

# Find the pattern: </button> followed by whitespace and )}
# Add </> between them
pattern = r'(</button>\r?\n)(\s+)(\)\})'
replacement = r'\1\2  </>\r\n\2\3'

new_content = re.sub(pattern, replacement, content, count=1)

with open(file_path, 'w', encoding='utf-8') as f:
    f.write(new_content)

print('Fixed: Added closing fragment tag </>')
